eigentrust/src/lib.rs

```diff
@@ -120,10 +121,17 @@ pub struct ClientConfig {
 	pub node_url: String,
 }
 
-/// Signer type alias.
-pub type ClientSigner = SignerMiddleware<Provider<Http>, LocalWallet>;
-/// Scores type alias.
-pub type Score = (Address, Scalar, RationalScore);
+/// Score struct.
+pub struct Score {
+	/// Participant address.
+	pub address: [u8; 20],
+	/// Scalar score.
+	pub score_fr: [u8; 32],
+	/// Rational score (numerator, denominator).
+	pub score_rat: ([u8; 32], [u8; 32]),
+	/// Hexadecimal score.
+	pub score_hex: [u8; 32],
```

Comment:
How are `score_fr` and `score_hex` different?

Reply:
- `score_hex` is the truncated integer representation of the `BigRational` (return of `converge_rational()`) expressed as a hex string. It could also be a String.
- `score_fr` is the scalar score, return value of `converge()`.

But it might be confusing, should I change the names of them? Or add some extra clarification maybe